### PR TITLE
feat: name transformations

### PR DESCRIPTION
**Pull request by will-larkin-nhs**
## Description
- added new queues to docker
- added ability to set cells to null with the parquet editor
- name transformation

## Context
[DTOSS-4285](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-4285)
[DTOSS-4286](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-4286)

## Type of changes

- [ ] Refactoring (non-breaking change)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [X] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [X] I have followed the code style of the project
- [X] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [X] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: application/CohortManager/src/Functions/Shared/Data/Database/BsTransformationLookups.cs
Comment (MWClayson-NHS) [inline application/CohortManager/src/Functions/Shared/Data/Database/BsTransformationLookups.cs:1]: Why are we calling the DB separately each of the fields here?

File: application/CohortManager/src/Functions/Shared/Data/Database/BsTransformationLookups.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    public string GetGivenName(string participantId)
+    {
+        return GetName(participantId, "GIVEN_NAME");
+    }
+
+    public string GetFamilyName(string participantId)
+    {
+        return GetName(participantId, "FAMILY_NAME");
+    }
+    public string GetName(string participantId, string nameType)
```
Comment (MWClayson-NHS) [inline application/CohortManager/src/Functions/Shared/Data/Database/BsTransformationLookups.cs:34]: Can you make this Private?

Comment (will-larkin-nhs) [reply application/CohortManager/src/Functions/Shared/Data/Database/BsTransformationLookups.cs]: yep, done

File: application/CohortManager/Set-up/azurite/parquet-editor.py [hunk 2]
```diff
@@ -93,6 +99,7 @@
     file_name = args.o[0]
 
 df.to_parquet(path=file_name, engine='fastparquet', index=False)
+# write(file_name, df, has_nulls=True)
```
Comment (andrewmorris43) [inline application/CohortManager/Set-up/azurite/parquet-editor.py:102]: Is this comment needed?

Comment (will-larkin-nhs) [reply application/CohortManager/Set-up/azurite/parquet-editor.py]: removed

**Review by MWClayson-NHS — APPROVED** [review]

**Pull request closed without merging**